admin_guide/ipsec.adoc

```diff
@@ -250,6 +250,11 @@ and begin encrypting:
 # systemctl enable ipsec
 ----
 
+[NOTE]
```

Comment:
It seems to, with the pre-req section having more detail. 
I can close this PR and leave the note out, if that makes sense.